libraries/device_shadow_demo_dependencies.cmake

```diff
@@ -17,7 +17,7 @@ endforeach()
 
 # Add cmake files of module to metadata.
 afr_module_cmake_files(${AFR_CURRENT_MODULE} 
-    ${CMAKE_CURRENT_LIST_DIR}/device_shadow_for_aws_iot_embedded_sdk/shadowFilePaths.cmake
```

Comment:
was this an issue in last release or caused by the merge of release?

Reply:
This is caused by the merger of the `release` branch into `master`